dbt_subprojects/daily_spellbook/models/addresses/chains/addresses_ethereum_info.sql

```diff
@@ -0,0 +1,24 @@
+{% set blockchain = 'ethereum' %}
+
+{{
+    config(
+        schema = 'addresses_' + blockchain,
+        alias = 'info',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['address'],
+        merge_update_columns = ['executed_tx_count', 'max_nonce', 'is_smart_contract', 'namespace', 'name', 'last_seen', 'last_tx_block_number'],
+    )
+}}
+
+{{
+    addresses_info(
+        blockchain = blockchain
+        , transactions = source(blockchain, 'transactions')
+        , token_transfers = source('tokens_' + blockchain, 'transfers')
+        , creation_traces = source(blockchain, 'creation_traces')
+        , first_funded_by = ref('addresses_events_' + blockchain + '_first_funded_by')
```

Comment:
`token_transfers` and `first_funded_by` seems unused in the macro.

Reply:
they were removed as asked by jeff, i'll add them back.